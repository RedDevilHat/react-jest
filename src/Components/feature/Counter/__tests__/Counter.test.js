import React from "react";

describe("Render Title component", () => {
    testRenderCounter();
    testValidCounter();
    testUnvalidCounter(undefined);
    testUnvalidCounter(null);
    testUnvalidCounter(NaN);
});

