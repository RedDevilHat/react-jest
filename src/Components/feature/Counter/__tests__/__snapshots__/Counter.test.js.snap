// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Title component render UI component 1`] = `
<div
  className="count"
>
  <span>
    Counter: 
  </span>
</div>
`;

exports[`Render Title component render UI component with UNVALID data 1`] = `
<div
  className="count"
>
  <span>
    Counter: 
  </span>
</div>
`;

exports[`Render Title component render UI component with UNVALID data 2`] = `
<div
  className="count"
>
  <span>
    Counter: 
  </span>
</div>
`;

exports[`Render Title component render UI component with UNVALID data 3`] = `
<div
  className="count"
>
  <span>
    Counter: 
    NaN
  </span>
</div>
`;

exports[`Render Title component render UI component with VALID data 1`] = `
<div
  className="count"
>
  <span>
    Counter: 
    999999
  </span>
</div>
`;
